### ISSUE TITLE
Extending picEdit

### ISSUE THREAD

**Issue by rainercedric23** [andyvr/picEdit#51]
Hi, is there some way to extend this library(ie. add error handler to current formSubmitted option)?

**Comment by dawondyifraw** [comment]
Yeah Sure - why don'y you try to look on picedit.js not min.js . it works with xmlhttp ajax - so kinda a hint
